frontends/web/src/components/forms/button.tsx

```diff
@@ -40,26 +47,25 @@ export function ButtonLink({
         ], className
     ].join(' ');
 
-    if (disabled) {
+    if (disabled === true) {
```

Comment:
why `=== true`? looks like the way it was before works the same way.

Reply:
No reason, actually I prefer `if (disabled) {` and will change it back